src/test/OptionSettlement.t.sol

```diff
@@ -47,6 +47,8 @@ contract OptionSettlementTest is Test, NFTreceiver {
     IERC20 public constant WETH = IERC20(WETH_A);
     IERC20 public constant USDC = IERC20(USDC_A);
 
+    uint40 public constant ONE_DAY_SECONDS = 24 * 60 * 60;
```

Comment:
Ditto previous comment, might as well just use `1 days` in-line imo

Reply:
good call